source/appModules/explorer.py

```diff
@@ -5,14 +5,15 @@
 #This file is covered by the GNU General Public License.
 #See the file COPYING for more details.
 
-"""App module for Windows Explorer (aka Windows shell).
+"""App module for Windows Explorer (aka Windows shell and renamed to File Explorer in Windows 8).
 Provides workarounds for controls such as identifying Start button, notification area and others.
 """
 
 from comtypes import COMError
 import time
 import appModuleHandler
 import controlTypes
+import winVersion

```

Comment:
This import isn't necessary -- winVersion is already imported a few lines below.